test/helpers/file.spec.js

```diff
@@ -68,3 +68,23 @@ describe('readFile', () => {
         }
     });
 });
+
+describe('splitExtension', () => {
```

Comment:
What is `async` in this tests?

Reply:
ouch, that's from copy and paste from the file array.spec.js (where I don't think it's needed either)... Could you hotfix them?